core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -124,18 +124,93 @@ describe('Exploration translation', function() {
     explorationEditorMainTab.setInteraction('Continue');
     responseEditor = explorationEditorMainTab.getResponseEditor('default');
     responseEditor.setDestination('final card', true, null);
-    // Setup a terminating state.
     explorationEditorMainTab.moveToState('final card');
     explorationEditorMainTab.setInteraction('EndExploration');
     explorationEditorMainTab.moveToState('first');
     explorationEditorPage.saveChanges();
-
```

Comment:
Keep the newline. It separates the exploration creation from the navigation.

Reply:
Making the change.